webpack/scenes/ContentViews/Table/Loading.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {
+  Bullseye,
+  Title,
+  EmptyState,
+  EmptyStateIcon,
+  Spinner,
+} from '@patternfly/react-core';
+
+const Loading = () => (
+  <Bullseye>
+    <EmptyState>
+      <EmptyStateIcon variant="container" component={Spinner} />
+      <Title size="lg">
+        Loading
+      </Title>
+    </EmptyState>
+  </Bullseye>
```

Comment:
Nice ! maybe we should move it into a general table helpers file

Reply:
The pf4 table and associated components will be abstracted in some way, ideally moved to foremanReact, its just a bit early to do that right now as there is no second use case. 

Perhaps I can move everything into a separate folder that is ready-to-be-moved as this feature progresses that will ensure things stay separate and uncoupled from a particular page. I just don't want to make too many assumptions too early.

I plan to do some refactoring along these lines in the next PR which adds pagination and search. This will give a clearer picture of how the table should be structured, and I can structure it appropriately. Does that sound ok?

Reply:
thanks, it can be in a follow up PR
though usually this `move_to` folders content stays there for a really long time
but this is the new content view.. so we'll see :)

Reply:
> thanks, it can be in a follow up PR
> though usually this `move_to` folders content stays there for a really long time
> but this is the new content view.. so we'll see :)

Great, I'll handle it in the upcoming PR. 

I don't think we should use `move_to` folders for anything, In general I am against that workflow and I don't think its a great habit to get into. 

Let me explain my plan a bit better:

1. In the upcoming PR adding search and pagination to the table, move any Table related components out of `scenes/ContentViews` to it's own separate folder in Katello and make sure its more general and able to be re-used. we'll have to think of a good name, but it should live in a general components folder that isn't a `move_to` one.
2. When we have a second use case, move the table to Foreman so it can be used with `foremanReact`.

I'm hesitant to move things without a second use case as any configurable parts we add to the table will be based on assumptions of what another page will use the table for. I would like to see a real-world use case first before we actually start designing the details of the Table. In my experience, abstracting too early leads to a lot of [YAGNI](https://martinfowler.com/bliki/Yagni.html) and pre-mature optimization. As far as I know, the CV page is the only one using the pf4 table right now (please let me know if I am missing something)

But for now, we can make sure the basic use cases are covered, things are kept modular, and there is a general basic design. Hopefully this makes sense :)

Reply:
Makes sense ! Thanks for explaining